src/components/ProtVista/Popup/Residue/index.js

```diff
@@ -29,25 +28,20 @@ const ProtVistaResiduePopup = ({ detail, sourceDatabase } /*: Props */) => {
   return (
     <section>
       <h6>
+        Residue in {sourceDatabase} -{' '}
         {accession.startsWith('residue:')
           ? accession.split('residue:')[1]
           : accession}
-        {description && <p>[{description}]</p>}
       </h6>
```

Comment:
I would remove the "Residue in ". Just above the track, there is the "Residues" title.